zml/tokenizer/hftokenizers/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "zml_tokenizer_hftokenizers"
+version = "0.1.0"
```

Comment:
is the Cargo stuff needed ?

I don't see bazel using the Cargo.toml file, is it read somehow ?